iroha/src/sumeragi.rs

```diff
@@ -225,11 +232,19 @@ impl Sumeragi {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use std::convert::TryInto;
+    use ursa::signatures::{ed25519::Ed25519Sha512, SignatureScheme};
 
     #[test]
     #[should_panic]
```

Comment:
Generally even running on 1 node (i.e., with no peers) should be fine. We just assume the user knows what they are doing and don't need the coolness of blockchain redundancy.

Reply:
I think it's more about @eadventurous changes. @eadventurous please create follow-up task

Reply:
Added task in JIRA https://soramitsu.atlassian.net/browse/RH2-72?atlOrigin=eyJpIjoiZGJkYzM1MWE5ZDRiNGQ5YjlmYTE0ZDY4MGVhYzI4NDUiLCJwIjoiaiJ9